tests/datasets/test_utils.py

```diff
@@ -2,17 +2,43 @@
 from czbenchmarks.datasets import utils
 import types
 from czbenchmarks.datasets.types import Organism
-from czbenchmarks.datasets.utils import load_dataset, load_local_dataset
+from czbenchmarks.datasets.utils import load_dataset, load_customized_dataset
 from unittest.mock import patch
 import pytest
 
 
-def test_load_local_dataset(tmp_path, monkeypatch):
-    """Test load_local_dataset instantiates and loads a dataset from a local file."""
-
-    # Create a dummy file to represent the dataset
-    dummy_file = tmp_path / "dummy.h5ad"
-    dummy_file.write_text("dummy content")
+# FIXME MICHELLE find class with data that loads more quickly
```

Comment:
TODO: will add an actual dataset that can use a simple h5ad that is created during the test to avoid file download issues.